### DISCUSSION TITLE
Rocket cluster program.

### DISCUSSION THREAD

**Discussion by amber-m-o-r-e** [rwf2/Rocket#2520] [Questions]
Hi,

I am willing to create Rocket cluster program, this cluster will be configurable at the start of the program. Any heads-up start or suggestion for this?

**Comment by SergioBenitez** [comment]
I'm not sure I totally understand what you're trying to create. Perhaps consider using [`main`](https://api.rocket.rs/v0.5-rc/rocket/attr.main.html) instead of `launch`, however. Or even [`execute()`](https://api.rocket.rs/v0.5-rc/rocket/fn.execute.html).
